Formula/doctest.rb

```diff
@@ -1,8 +1,8 @@
 class Doctest < Formula
   desc "Feature-rich C++11/14/17/20 single-header testing framework"
   homepage "https://github.com/onqtam/doctest"
```

Comment:
May want to also update homepage to new org redirect:
```suggestion
  homepage "https://github.com/doctest/doctest"
```